hack/common.sh

```diff
@@ -2,14 +2,18 @@
 
 set -e
 
-pushd .
+pushd . >&2
 cd "$(dirname "$0")/.."
 
 function finish {
-    popd
+    popd >&2
 }
 trap finish EXIT
 
+function debug {
+    echo "$@" >&2
```

Comment:
can you do `echo "DEBUG: @0"`
so we can easily filter and distinguish between debug and errors?